src/Microsoft.Restier.WebApi/ODataDomainController.cs

```diff
@@ -50,7 +50,7 @@ protected ODataDomainController()
         }
 
         /// <summary>
-        /// Gets the <see cref="IDomain"/> interface of the domain of the <see cref="ODataDomainController"/>.
```

Comment:
@karataliu suggested that "Gets the interface of ..." is not proper so change to this statement.
P.S., this comment was originally not present and added by me several days ago. So it's OK to change the way expressing.